.github/workflows/ci.yml

```
@@ -8,8 +8,11 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - php: '7.4'
+          - php: '8.1'
```

Based on the last test, Moodle is definitely not ready for PHP 8.1 at this moment.
```suggestion
          - php: '8.0'
```

---

Worth a shot